addons/index.md

```diff
@@ -7,16 +7,52 @@ layout: documentation
+This includes all new 2.0 bindings as well as all 1.x add-ons
+that were reported to be compatible.
 Add-ons can be easily installed through the Paper UI (menu entry "Add-ons").
 
+| Add-on Type                             | Description                                                                                                               |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| [Bindings](bindings.html)               | Bindings integrate physical hardware, external systems and web services in openHAB                                        |
+| [User Interfaces](uis.html)             | User interfaces are either native smartphone apps or web applications that access the openHAB server through the REST API |
+| [Persistence](persistence.html)         | Persistence services allow openHAB to store time series data for history-based actions or statistics                      |
+| [Actions](actions.html)                 | Actions are predefined methods for openHAB rules and scripts                                                              |
+| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items                               |
+| [Voice Services](voices.html)           | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc.                                |
+| [3rd Party System Integration](io.html) | Expose openHAB to external systems                                                                                        |
 
-| Add-on Type | Description |
-|-------------|----------------------|
-| [Bindings](bindings.html) | Bindings integrate physical hardware, external systems and web services in openHAB |
-| [User Interfaces](uis.html) | User interfaces are either native smartphone apps or web applications that access the openHAB server through the REST API |
-| [Persistence](persistence.html) | Persistence services allow openHAB to store time series data for history-based actions or statistics |
-| [Actions](actions.html) | Actions are predefined methods for openHAB rules and scripts |
-| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items |
-| [Voice Services](voices.html) | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc. |
-| [3rd Party System Integration](io.html) | Expose openHAB to external systems |
+## Installation of Add-ons
+
+Add-ons can be installed in different ways.
+
+### Through Paper UI
+
+Navigate to the add-ons section.
+Search for the needed add-on in the categories and press install.
+
+![installing Add-ons through Paper UI](images/index_installation_paperui.gif){: width="720px"}
+
+### From the Karaf Console
```

Comment:
Console or Remote Console. It was decided at some point to not expose "Karaf" to the end user.

Reply:
@ThomDietrich I missed that we changed the name of Karaf console to Remote Console.  Just curious about the thinking, and should this be mentioned somewhere in the intro of the docs?  ISTM that if someone does not know what a Remote Console is, they can't google it and come up with info.  But googling Karaf gets a lot of useful info.  Just mentioning this because that is exactly what I did when I first encountered it.

Reply:
Karaf is a technology/software used by openHAB/ESH. Someone whos looking for information about the openHAB (Karaf) Console will not be helped with a "Karaf console how to list items" Google search. The Console should primarily be seen as the "openHAB Console".

The fact that the openHAB console is backed by Karaf is written here: https://docs.openhab.org/administration/index.html I'm not saying that it's perfectly documented but Hey, feel free to improve ;)

Reply:
Yea i agree fully in this point.
I just didn't know it. Will change it.

Reply:
BTW:

I would go for **openHAB console** then.

Wdyt?

Reply:
+1 openHAB Console

Reply:
Yup fine with me.

Reply:
I wasn't aware of this either, but it sounds reasonable to me 😆